scout/constants/variant_tags.py

```diff
@@ -32,6 +32,7 @@
     "genetic_models",
     "acmg_classification",
     "genes",
+    "hgnc_symbols",
```

Comment:
I hate this change we introduced some time ago (https://github.com/Clinical-Genomics/scout/pull/2861). Every time something is missing from general report but should be there it takes me forever to figure out why! I wish we could revert 😠

Reply:
Heh, yes, I guess the react version of the report is still on hold, so perhaps the use case isn't really there?